activestorage/test/analyzer/video_analyzer_test.rb

```diff
@@ -13,7 +13,7 @@ class ActiveStorage::Analyzer::VideoAnalyzerTest < ActiveSupport::TestCase
     assert_equal 640, metadata[:width]
     assert_equal 480, metadata[:height]
     assert_equal [4, 3], metadata[:display_aspect_ratio]
-    assert_equal 5.166648, metadata[:duration]
+    assert_equal true, metadata[:duration].between?(4, 6)
```

Comment:
This assertion is not failed even without this PR since both ffprobe version (3.2.14 and 4.1.3) returns the same duration 5.166648 for `video.mp4`.
So there is no need to touch this assertion in this point.

Reply:
Your original commit comment was "Fix activestorage CI failure due to ffprove version differece", is that not the case?

Reply:
That commit is to fix CI failure for `rotated_video.mp4`.
You can try to confirm no difference by this branch #36695 in your locally.